### PR TITLE
chore(docker/ci): updating to use node 14 and postgres 13

### PR DESCRIPTION
**Pull request by WikiRik**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 version: 2.1
 jobs:
   test:
     machine:
-      image: ubuntu-1604:201903-01
+      image: ubuntu-2004:202010-01
     steps:
       - checkout
-      - run: docker run --name postgres -p 5432:5432 -d postgres:10.6
+      - run: docker run --name postgres -e POSTGRES_PASSWORD='5ecr3t' -p 5432:5432 -d postgres:latest
```
Comment (serge1peshcoff) [inline .circleci/config.yml:10]: do we need this one?

Comment (WikiRik) [reply .circleci/config.yml:10]: Postgres in general or the password?

Comment (WikiRik) [reply .circleci/config.yml:10]: If the answer is password, then yes. It is mandatory now

Comment (serge1peshcoff) [reply .circleci/config.yml:10]: password, yes

Comment (serge1peshcoff) [reply .circleci/config.yml:10]: also afaik it's better to pin packages versions, so we won't break if there'd be a semver upgrade

Comment (WikiRik) [reply .circleci/config.yml]: I've pinned it to 13

**Review by serge1peshcoff — APPROVED** [review]

**Pull request merged**